test/javascript.py

```diff
@@ -7,7 +7,10 @@
 
 
 def javascript_tests():
-    if os.path.exists(karma):
+    if not shutil.which('nodejs'):
```

Comment:
Is there a scenario where "node_modules/.bin/karma" exists where "nodejs" doesn't? When installing npm, Debian seems to be installing nodejs too.

Reply:
the scenario that I hit, which is I had this stuff installed in my host machine, but not in the container created by dev-docker

(I tried to add nodejs etc to it, but chromium crashes weirdly inside docker and I didn't want to investigate that at the moment)

Reply:
(just saw you other comment)